### PR TITLE
change newfolder link to button

### PR DESCRIPTION
**Pull request by djamnazi**
[OO-FEATURE] change new link to button

### CODE REVIEW

File: MediaAdminBundle/Resources/views/Tree/menu_macros_bo.html.twig [hunk 1]
```diff
@@ -17,8 +17,10 @@
                 data-url="{{ path('open_orchestra_media_admin_folder_new', {'parentId': parentId}) }}"
                 data-type="folder"
             >
+                <span class="btn btn-success">
                 <i class="fa fa-file"></i>
```
Comment (veolys-nanne) [inline MediaAdminBundle/Resources/views/Tree/menu_macros_bo.html.twig]: add indent before <i>

File: MediaAdminBundle/Resources/views/Tree/menu_macros_modal.html.twig [hunk 2]
```diff
@@ -16,8 +16,10 @@
                href="#"
                data-url="{{ path('open_orchestra_media_admin_folder_new', {'parentId': parentId}) }}"
            >
+           <span class="btn btn-success">
            <i class="fa fa-file"></i>
```
Comment (veolys-nanne) [inline MediaAdminBundle/Resources/views/Tree/menu_macros_modal.html.twig]: same here

**Comment by djamnazi** [comment]
@itkg-nanne PR updated

File: MediaAdminBundle/Resources/views/Tree/menu_macros_modal.html.twig [hunk 3]
```diff
@@ -16,8 +16,10 @@
                href="#"
                data-url="{{ path('open_orchestra_media_admin_folder_new', {'parentId': parentId}) }}"
            >
-           <i class="fa fa-file"></i>
-                    {{ 'open_orchestra_media_admin.left_menu.tree.new_media_folder'|trans }}
+                   <span class="btn btn-success">
+                        <i class="fa fa-file"></i>
+                            {{ 'open_orchestra_media_admin.left_menu.tree.new_media_folder'|trans }}
+                   </span>
```
Comment (ngilain) [inline MediaAdminBundle/Resources/views/Tree/menu_macros_modal.html.twig]: You have an ident problem the closing a, div and li does not have the same padding as the opening tags

**Comment by alavieille** [comment]
:+1:  but can you add the tag [OO-BUGFIX] by [OO-FEATURE] with a comment to explain the feature and add it in the first comment of your pull request.

**Comment by ngilain** [comment]
:+1: but as @itkg-alavieille said, the tag [OO-FEATURE] should be added in the PR comment zone (the one in the top where I wrote "Put your tag here djamnazi" :)

**Pull request merged**